### DISCUSSION TITLE
Welcome to pwvucontrol Discussions!

### DISCUSSION THREAD

**Discussion by saivert** [saivert/pwvucontrol#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GreyXor** [comment]
Hi thanks, nice to see a modern pavucontrol alternative with rust and gtk4. Thanks @saivert.
As soon as it's wireplumber 0.5 compatible, I'll make a `pwvucontrol-git` package on https://aur.archlinux.org/

**Comment by luyu-wu** [comment]
Heya! I've been following this project for a while now. As Xor said, it's great to see a modern and lightweight pavucontrol alternative!
I may contribute from time to time but I'm unfortunately pretty tied up with school and some coompetitions right now.

Thanks for the amazing work!

**Comment by wildwestrom** [comment]
Bravo! This is fucking awesome!
